src/pudl/etl.py

```diff
@@ -15,326 +15,187 @@
-
-    ds = Datastore(**ds_kwargs)
-    # Extract EIA forms 923, 860
-    eia923_raw_dfs = pudl.extract.eia923.Extractor(ds).extract(
-        year=eia923_years)
-    eia860_raw_dfs = pudl.extract.eia860.Extractor(ds).extract(
-        year=eia860_years)
-    # if we are trying to add the EIA 860M YTD data, then extract it and append
-    if eia860m:
-        eia860m_raw_dfs = pudl.extract.eia860m.Extractor(ds).extract(
-            year_month=pc.WORKING_PARTITIONS['eia860m']['year_month'])
-        eia860_raw_dfs = pudl.extract.eia860m.append_eia860m(
-            eia860_raw_dfs=eia860_raw_dfs, eia860m_raw_dfs=eia860m_raw_dfs)
-
-    # Transform EIA forms 923, 860
-    eia860_transformed_dfs = pudl.transform.eia860.transform(
-        eia860_raw_dfs, eia860_tables=eia860_tables)
-    eia923_transformed_dfs = pudl.transform.eia923.transform(
-        eia923_raw_dfs, eia923_tables=eia923_tables)
-    # create an eia transformed dfs dictionary
-    eia_transformed_dfs = eia860_transformed_dfs.copy()
-    eia_transformed_dfs.update(eia923_transformed_dfs.copy())
-
-    # convert types..
-    eia_transformed_dfs = pudl.helpers.convert_dfs_dict_dtypes(
-        eia_transformed_dfs, 'eia')
-
-    entities_dfs, eia_transformed_dfs = pudl.transform.eia.transform(
-        eia_transformed_dfs,
-        eia860_years=eia860_years,
-        eia923_years=eia923_years,
-        eia860m=eia860m,
-    )
-    # convert types..
-    entities_dfs = pudl.helpers.convert_dfs_dict_dtypes(entities_dfs, 'eia')
-    for table in entities_dfs:
-        entities_dfs[table] = PUDL_META.get_resource(table).encode(entities_dfs[table])
-
-    out_dfs.update(entities_dfs)
-    out_dfs.update(eia_transformed_dfs)
-    return out_dfs
+def command_line_flags() -> argparse.ArgumentParser:
+    """Returns argparse.ArgumentParser containing flags relevant to the ETL component."""
+    parser = argparse.ArgumentParser(
+        description="ETL configuration flags", add_help=False)
+    parser.add_argument(
+        "--executor",
+        choices=["LOCAL", "LOCAL-DASK", "DASK"],
+        default="LOCAL",
+        help="Which Prefect executor to run the ETL on."
+    ),
+    parser.add_argument(
+        "--dask-executor-address",
+        default=None,
+        help='If specified, use pre-existing DaskExecutor at this address.')
+    # TODO(bendnorman): Should upload-to be supported right now?
+    parser.add_argument(
+        "--upload-to",
+        type=str,
+        default=os.environ.get('PUDL_UPLOAD_TO'),
+        help="""A location (local or remote) where the results of the ETL run
+        should be uploaded to. This path will be interpreted by fsspec so
+        anything supported by that module is a valid destination.
+        This should work with GCS and S3 remote destinations.
+        Default value for this will be loaded from PUDL_UPLOAD_TO environment
+        variable.
+        Files will be stored under {upload_to}/{run_id} to avoid conflicts.
+        """)
```

Comment:
Do we want to support upload_to right now?

Reply:
That is true. If we aren't running this remotely soon should it be included in this PR? I don't want to add functionality that isn't fully supported.